docs/layouts/shortcodes/generated/vector_search_runtime_config_configuration.html

```diff
@@ -0,0 +1,36 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>async</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>Boolean</td>
+            <td>Value can be 'true' or 'false' to suggest the planner choose the corresponding predict function. If the backend search function provider does not support the suggested mode, it will throw exception to notify users.</td>
+        </tr>
+        <tr>
+            <td><h5>max-concurrent-operations</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>Integer</td>
+            <td>The max number of async i/o operation that the async ml predict can trigger.</td>
```

Comment:
Mention default value?
```suggestion
            <td>The max number of async i/o operation that the async vector search can trigger.</td>
```